src/Parser/Core/Modules/Items/Healthstone.js

```diff
@@ -0,0 +1,42 @@
+import SPELLS from 'common/SPELLS';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Abilities from 'Parser/Core/Modules/Abilities';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+
+const HEALTHSTONE_SPELLS = [
```

Comment:
Could this or prepotion somehow work together? I don’t want to have to maintain two identical lists.

Reply:
HS/healthpots don't share cooldowns with DPS/mana pots. Shouldn't be any overlaps in the lists.